src/style.css

```diff
@@ -0,0 +1,132 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  --bloks_width: 166px;
+  --main-text_color: #060b35;
+  --minor-text_color: #616070;
+}
+
+.card {
+  display: flex;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+}
+
+.card__text {
+  font-weight: 500;
+  font-size: 12px;
+  color: var(--main-text_color);
+  width: var(--bloks_width);
+  line-height: 18px;
+  margin-top: 40px;
+}
+
+.card__product-code {
+  display: inline-block;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--minor-text_color);
+  left: 16px;
+  text-align: left;
+  width: var(--bloks_width);
+  margin-top: 4px;
+}
+
+.card__assessment {
+  display: flex;
+  width: var(--bloks_width);
+  justify-content: space-between;
+  margin-top: 16px;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__assessment_reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  display: inline;
+}
+
+.card__cost {
+  display: flex;
+  width: var(--bloks_width);
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 23px;
+}
+
+.card__cost_text {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--minor-text_color);
+}
+
+.card__cost_price {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--main-text_color);
+}
+
+.card__button {
+  text-decoration: none;
+  text-transform: uppercase;
+  width: 166px;
+  height: 40px;
+  text-align: center;
+  color: #fff;
+  background-color: #00acdc;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  font-weight: 700;
+  font-size: 14px;
+  margin-top: 16px;
+  margin-bottom: 16px;
+}
+
+.card__button:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
+  width: 165px;
+  height: 39px;
```

Comment:
better use invisible border on main element and make it visible on hover (or vice versa)
do not forget about `border-box`